Dockerfile

```diff
@@ -105,7 +105,9 @@ RUN jupyter nbextension enable --py widgetsnbextension
 # https://github.com/joblib/joblib/blob/0.11/joblib/parallel.py#L328L342
 ENV JOBLIB_TEMP_FOLDER=/tmp
 
+# Filter out know irrelevant warnings.
```

Comment:
could you please add a TODO comment here saying that this can be removed when TF is upgraded?

Reply:
Done.